.gitignore

```diff
@@ -1,17 +1,9 @@
-# The directory Mix will write compiled artifacts to.
```

Comment:
Is there a reason to remove comments on this file? They add clarification to newcomers, and they don't cost anything.